R/dl_build_src_gran.R

```diff
@@ -22,18 +23,27 @@ dl_build_src <- function(GRAN.dir = './GRAN'){
 	##Update all the local packages so we're always working with the latest
 	update.packages(ask=FALSE, lib.loc = Sys.getenv('R_LIBS_USER'), repos=repos)
 	
-	packages = read_src_list()
+	#current packages on GRAN
+	currentListPath <- "./granCurrent.tsv"
+	packages = read_src_list(currentListPath)
+	
+	#unlink(file.path(GRAN.dir, 'src'), recursive=TRUE)   #need to delete only updated directories
+	#dir.create(src_dir, recursive = TRUE)
+	toDelete <-  sub(".*\\/","",packages$package)
+	if(dir.exists(file.path(GRAN.dir, 'src'))){
+	  system(paste("rm",paste0(paste0(src_dir,"/",toDelete,"*"),collapse=" "))) #delete any existing version of toDelete packages
```

Comment:
rm is a *nix and macOS command line tool. Likely not available on Win. Can you use 'unlink' R command instead?